docs/index.html

```diff
@@ -121,15 +120,11 @@ <h3 class="title center">Koding can provision dev environments on your favorite
 </div>
 
 <div class='Homepage-use-cases Homepage--containerOffset'>
-  <h4 class="title center">GET STARTED WITH KODING</h4>
-  <h3 class="title center">Koding takes care of many other use cases too.</h3>
+  <h4 class="title center">YOUR <span>CUSTOMERS</span> BENEFIT FROM KODING TOO</h4>
+  <h3 class="title center">Solving your software's sales and training issues</h3>
   <div class='Homepage--container'>
-    <h6>Creating consistent environments that are instantly shareable helps your organization in multiple ways.</h6>
-    <div class="actions">
-      <a class="cta" href="{{ site.url }}/teams/create">Start Free Trial</a>
-      <a class="cta secondary" href="{{ site.url }}/demo">Request a Demo</a>
-    </div>
-    <hr>
+    <h6>You have a "Download" link that everyone clicks, yet vital customers are lost through your setup instructions. Let's help you engage with them!</h6>
+    <div class="cases-title">You can use Koding for...</div>
```

Comment:
get rid of `...` but use a `:`.
Also, a note for the future, use ellipsis `…` instead of three dots `...` where necessary.